### ISSUE TITLE
teensy 4.1 usb client wires pads

### ISSUE THREAD

**Issue by snowsh01** [ElectroTechnique/TSynth-Teensy4.1#13]
Note to be extra careful when soldering the USB jumper wires to the Teensy 4.1 D+ and D- pads. Compared to previous Teensy boards, the 4.1 pads are tiny and very weak. any movement of the wires will likely result in the pads pulling off. 

Looking at the Teensy 4.1 schematics a there is fix in the event this occurs but I am yet to perform the fix. 

Fix:

If you have damaged the pads, you could connect D+ and D- via U5 pins 6 and 5 respectively

https://www.pjrc.com/teensy/schematic.html

![parts_placement_teensy41_bottom](https://user-images.githubusercontent.com/78615935/107118187-4b615900-6877-11eb-9368-b2bdbc518711.png)
![parts_placement_teensy41_top](https://user-images.githubusercontent.com/78615935/107118189-4bf9ef80-6877-11eb-9641-a44a2d6ba354.png)
![schematic41](https://user-images.githubusercontent.com/78615935/107118191-4c928600-6877-11eb-91b4-96b1c197a54d.png)

**Comment by philippejadin** [comment]
I destroyed mine quickly indeed. I'd advice to use very thin wire, solder them like shown in the user guide, and right after put a drop of hot glue to protect the pads and the wires. Too late for me :-/

**Comment by snowsh01** [comment]
Yep. I killed mine and learnt the hard way.... Unfortunately there seems to be no easy way to solder in elsewhere

**Comment by philippejadin** [comment]
It's not that bad, you can use the micro usb instead for conencting and programming, and if/when it get destroyed, you can unsolder it, and connect back the D+ D- wires from underneath the connector. I would only do it if the micro usb got destroyed because it's probably hard surgery to remove the micro usb connector without touching the rest. It's imho a negative point of teensy : they are too small :-)

**Comment by antoniograzioli** [comment]
too late for me as well, i soldered them , they were working, but after some rremoval/reinserts of the teensy they fell apart...
